_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
+description: Research Page
+tags: [robotics,panda,simulator,research]
+permalink: robotic_skin.html
+---
+
+{% include image.html url="research/skin_unit.png" max-width="100%" description="Flexible skin unit created by etching the circuit diagram into a copper sheet and then encasing it in the flexible polymer ecoflex." %}
+
+
+Robots have been transitioning into human-populated environments and replacing their physical cages used to keep humans safe with complex perception and software. However, current solutions are computationally expensive, prone to occlusion, and require setup overhead. Robots require compact, self-contained sensing of nearby space to provide inherently save interactions. In this work, we present the first prototype of a flexible, conformant whole-body artificial skin for collaborative robotics. In order to locate all skin units on the surface of a robot, a novel kinematically calibration algorithm is required to reduce setup time.
```

Comment:
> "In order to locate all skin units on the surface of a robot, a novel kinematically calibration algorithm is required to reduce setup time."

This comes out of nowhere and needs explanation. Nobody can understand how you jump from the previous sentence to this.